src/main/java/io/confluent/connect/elasticsearch/ElasticsearchClient.java

```diff
@@ -663,17 +638,17 @@ private boolean handleMalformedDocResponse(BulkItemResponse response) {
         return false;
       case FAIL:
```

Comment:
nit. do we need `case FAIL:`?

Reply:
Yes, it is not needed, but since this is part of old code, I didn't modify it.